variables.tf

```diff
@@ -146,3 +146,9 @@ variable "nlb_accounts" {
   default     = []
   type        = "list"
 }
+
+variable "cloudtrail_sse_options" {
```

Comment:
I was confused by the `options` naming. Aren't you implementing these as requirements and thus the name should be something like  `cloudtrail_sse_requirements`?